### DISCUSSION TITLE
Can't Locate the Kovan Test Network

### DISCUSSION THREAD

**Discussion by Tarreef** [Cyfrin/foundry-full-course-cu#2745] [Q&A]
in the Course, it recommends switching to the kovan network for the price Consumer reference Contract but i can't locate the kovan network in metamesk let alone switching.
i have searched the github tab for where it indicates the network to use but i can't find it.
can someone help please

**Comment by EngrPips** [comment]
Hello @Tarreef, Kovan is now deprecated please use `Sepolia` instead

**Reply by Tarreef** [reply]
Sorry but please, is it just sepolia or ZK sync Sepolia??

**Reply by EngrPips** [reply]
`Sepolia`, when you get to the part where you will use `zksync sepolia` the instructor will inform you.
